src/main/java/me/tongfei/progressbar/ProgressBar.java

```diff
@@ -99,20 +103,24 @@ public ProgressBar(
      * @param updateIntervalMillis Update time interval (default value 1000ms)
      * @param renderer Progress bar renderer
      * @param consumer Progress bar consumer
+     * @param startFrom Initial completed process value
+     * @param elapsedSecond Initial elapsed second before
      */
     public ProgressBar(
             String task,
             long initialMax,
             int updateIntervalMillis,
             ProgressBarRenderer renderer,
-            ProgressBarConsumer consumer
+            ProgressBarConsumer consumer,
+            long startFrom,
+            long elapsedSecond
```

Comment:
I would prefer moving `long startFrom, Duration elapsed` before the renderer and the consumer. The arguments are grouped into 3 groups semantically: the progress state, the renderer, and the consumer.